cocos/renderer/backend/metal/TextureMTL.mm

```diff
@@ -376,5 +387,13 @@ bool convertData(uint8_t* src, unsigned int length, TextureFormat format, uint8_
     [commandBuffer commit];
 }
 
+void TextureCubeMTL::generateMipmaps()
+{
+    if (TextureUsage::RENDER_TARGET == _textureUsage)
+        return;
+    
+    Utils::generateMipmaps(_mtlTexture);
+    _isMipmapEnabled = true;
```

Comment:
There is an optimization: don't have to generate mipmaps if it is generated before. Of course, if the data is updated, then need to re-generate mipmap again.